release-notes/6.0/6.0.0.md

```diff
@@ -0,0 +1,141 @@
+# Release Notes
+
+## [Stable release 6.0.0] - 2024-12-09
+
+This update brings the below changes over the previous stable release:
+
+### Contributors
+Many thanks to the following public contributors! Their efforts toward furthering this project are very much appreciated!
+
+- [0xced](https://github.com/0xced)
+- [deadlydog](https://github.com/deadlydog)
+- [EamonHetherton](https://github.com/EamonHetherton)
+- [edwardneal](https://github.com/edwardneal)
+- [ErikEJ](https://github.com/ErikEJ)
+- [Markeli](https://github.com/Markeli)
+- [MichelZ](https://github.com/MichelZ)
+- [mus65](https://github.com/mus65)
+- [SimonCropp](https://github.com/SimonCropp)
+- [TrayanZapryanov](https://github.com/TrayanZapryanov)
+- [twsouthwick](https://github.com/twsouthwick)
+- [wilbit](https://github.com/wilbit)
+- [Wraith2](https://github.com/Wraith2)
+
+### Breaking Changes
+
+- Removed support for .NET Standard. [#2386](https://github.com/dotnet/SqlClient/pull/2386), since Preview 1
```

Comment:
I am also aligned with @ErikEJ that the 'since preview...' are not really needed for end users. The release notes for past previews should continue to stay in Changelog if we want to find their references. I haven't seen any other .NET product/library publish this way.

Reply:
Agreed.  The phrase "This update brings the below changes over the previous stable release" already makes it clear that the change summary is a delta from the previous stable release.  Knowing which preview a change happened to appear first in doesn't seem helpful, and users could easily find it by looking at the preview release notes.  Additionally, some changes in this new stable release may not have appeared in any preview.